src/pages/jihong/Main/Main.js

```diff
@@ -1,9 +1,45 @@
-import React from 'react';
-import './Main';
+import React, { useState, useEffect } from 'react';
 import Nav from '../../../components/Nav/Nav';
+import Aside from './Aside/Aside';
+import Feed from './Feed/Feed';
+import '../../../common.scss';
+import './Main.scss';
 
 const Main = () => {
-  return <Nav />;
+  const [feedList, setFeedList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/feedJihongData.json', {
+      method: 'GET', // GET method는 기본값이라서 생략이 가능합니다.
+    }) // 예시코드에서는 이해를 돕기 위해 명시적으로 기입해뒀습니다.
+      .then(res => res.json())
+      .then(data => {
+        setFeedList(data);
+      });
+  }, []);
+
+  return (
+    <div className="Container">
+      <Nav />
```

Comment:
Main과 관련 없는 Nav 컴포넌트이기 때문에, Main 컴포넌트의 css를 적용시키지 않게 하려면 밖으로 빼는 것이 좋습니다.
```suggestion
    <>
      <Nav />
      <div className="Container">
```